index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
Theres a bit of inconsistency regarding indentation, most of the project has 2 spaces but the HTML has 4. Could this be made 2 to match with everything else?